packages/ubuntu_provision/assets/ubuntu-provision.yml

```diff
@@ -1,6 +1,39 @@
-# This file is only to be used for tests
-scope:
-  test:
+pages:
+  locale:
     title: "Welcome"
-    image: "mascot.png"
-    visible: true
\ No newline at end of file
+    image: "packages/ubuntu_provision/assets/mascot.png"
```

Comment:
Maybe it'd also be worth making `packages/ubuntu_provision` the default for asset images, so that we can just specify `image: assets/mascot.png` here?